packages/components/dropdown/src/dropdown.vue

```diff
@@ -148,12 +147,9 @@ export default defineComponent({
     const scrollbar = ref(null)
     const currentTabId = ref<string | null>(null)
     const isUsingKeyboard = ref(false)
-    const triggerKeys = [
-      EVENT_CODE.enter,
-      EVENT_CODE.numpadEnter,
-      EVENT_CODE.space,
-      EVENT_CODE.down,
-    ]
+    const dropdownTriggerKeys = computed(() =>
+      isArray(props.triggerKeys) ? props.triggerKeys : []
+    )
```

Comment:
Why do we need to write it this way? It looks like redundant code

Reply:
I originally wanted to ensure that even if the format passed in by the user is incorrect, the format passed into the tooltip will not be wrong.

Reply:
`type: Array` 

In that case, Vue will also trigger an error. We do not seem to have to worry about such things happening

Reply:
Ok, let's remove the unnecessary logic. @hanchao-c0ldwave

Reply:
I have already removed it.